src/API/Google/Settings.php

```diff
@@ -349,6 +350,69 @@ protected function get_store_state(): string {
 		return $wc->get_wc_countries()->get_base_state();
 	}
 
+	/**
+	 * Get the WooCommerce store physical address.
+	 *
+	 * @return AccountAddress
+	 *
+	 * @since x.x.x
+	 */
+	public function get_store_address(): AccountAddress {
+		/** @var WC $wc */
+		$wc = $this->container->get( WC::class );
```

Comment:
For other classes where we get a dependency from the container we add it in a [comment up top](https://github.com/woocommerce/google-listings-and-ads/blob/1.3.0/src/MerchantCenter/MerchantCenterService.php#L25-L32) so we don't have to look through the code to find them. Eventually we might want to refactor this class to either be ContainerAware or get the dependencies directly, but I guess we have #563 for that.

Reply:
Yeah, it should eventually be refactored that's why I didn't make any more changes than I had to. The `WC` class was already being used in this service so this isn't a new addition.